example/spec/helpers/kinesisHelpers.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+const _ = require('lodash');
+const { Kinesis, StepFunctions } = require('aws-sdk');
+
+const {
+  LambdaStep,
+  getWorkflowArn,
+  timeout
+} = require('@cumulus/integration-tests');
+
+const { loadConfig } = require('../helpers/testUtils');
+
+const testConfig = loadConfig();
+
+const lambdaStep = new LambdaStep();
+const sfn = new StepFunctions({ region: testConfig.awsRegion });
+const kinesis = new Kinesis({ apiVersion: '2013-12-02', region: testConfig.awsRegion });
```

Comment:
@laurenfrederick Should these be pushed into `@cumulus/common/aws`.  I thought of that yesterday when marc was walking us though his code.

Reply:
I think all of these kinesis helpers should go in @cumulus/common/aws - but would be fine if we added a ticket to do that later.

Reply:
Yeah that is an excellent point 👍 If you want to merge to get unblocked for the next ticket, you could do it there as well.